tools/example_scripts/MakeNiiFilesBIDSCompliant.pl

```diff
@@ -0,0 +1,258 @@
+#! /usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Tabular;
+use File::Path qw/ make_path /;
+use File::Basename;
+use NeuroDB::DBI;
+
+my $profile = undef;
+my $tarchiveID = undef;
+
+my @opt_table = (
+    [ "-profile", "string", 1, \$profile,
+      "name of config file in ../dicom-archive/.loris_mri"
+    ],
+    [ "-tarchive_id", "string", 1, \$tarchiveID,
+      "tarchive_id of the .tar to be processed from tarchive table"
+    ]
+); 
+
+my $Help = <<HELP;
+
+This script will remove the root directory from the ArchiveLocation field
+in the tarchive table to make path to the tarchive relative. This should 
+be used once, when updating the LORIS-MRI code.
+
+HELP
+
+my $Usage = <<USAGE;
+
+Usage: $0 -help to list options
+
+USAGE
+
+&Getopt::Tabular::SetHelp($Help, $Usage);
+&Getopt::Tabular::GetOptions(\@opt_table, \@ARGV) || exit 1;
+
+################################################################
+################### input option error checking ################
+################################################################
+{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
+if ($profile && !@Settings::db) {
+    print "\n\tERROR: You don't have a configuration file named ".
+          "'$profile' in:  $ENV{LORIS_CONFIG}/.loris_mri/ \n\n";
+    exit 2;
+}
+
+################################################################
+######### Establish database connection ########################
+################################################################
+my $dbh = &NeuroDB::DBI::connect_to_db(@Settings::db);
+print "\n==> Successfully connected to database \n";
+
+################################################################
+#### This setting is in the ConfigSettings table   #############
+################################################################
+my $dataDir = &NeuroDB::DBI::getConfigSetting(
+                            \$dbh,'dataDirBasepath'
+                            );
+my $prefix = &NeuroDB::DBI::getConfigSetting(
+                            \$dbh,'prefix'
+                            );
+$dataDir    =~ s/\/$//g;
+
+# Make destination directory for the NII files
+# same level as assmebly/ directory but named as BIDS/
+my $destDir = $dataDir . "/BIDS";
+make_path($destDir) unless(-d  $destDir);
+
+################################################################
+# Grep tarchive list in a hash                          ########
+# %tarchive_list = {                                    ########
+#      $TarchiveID => {                                 ########
+#          'ArchiveLocation'    => $ArchiveLocation     ########
+#          'NewArchiveLocation' => $newArchiveLocation  ########
+#      }                                                ########
+# };                                                    ########
+################################################################
```

Comment:
clearly copied and pasted the script from another script of mine! I need to change this documentation.